src/android/hooks/configure-android.js

```diff
@@ -34,21 +38,22 @@ module.exports = function (ctx) {
     
     var replaceWith = getConfidId(data) + ".R";
     
+    const appSrcJavaDir = 'app/src/main/java';
     var platformRoot = path.join(ctx.opts.projectRoot, 'platforms/android');
     var fileImportR = [
-		{filePath: 'src/com/radaee/cordova/RadaeePDFPlugin.java', importStatement: 'com.radaee.viewlib.R'},
-    	{filePath: 'src/com/radaee/pdf/Global.java', importStatement: 'com.radaee.viewlib.R'},
-    	{filePath: 'src/com/radaee/reader/PDFLayoutView.java', importStatement: 'com.radaee.viewlib.R'},
-    	{filePath: 'src/com/radaee/reader/PDFNavAct.java', importStatement: 'com.radaee.viewlib.R'},
-    	{filePath: 'src/com/radaee/reader/PDFViewAct.java', importStatement: 'com.radaee.viewlib.R'},
-    	{filePath: 'src/com/radaee/reader/PDFViewController.java', importStatement: 'com.radaee.viewlib.R'},
-		{filePath: 'src/com/radaee/reader/PDFPagerAct.java', importStatement: 'com.radaee.reader.R'},
-    	{filePath: 'src/com/radaee/util/OutlineListAdt.java', importStatement: 'com.radaee.viewlib.R'},
-    	{filePath: 'src/com/radaee/util/PDFGridItem.java', importStatement: 'com.radaee.viewlib.R'},
-		{filePath: 'src/com/radaee/util/PopupEditAct.java', importStatement: 'com.radaee.viewlib.R'},
-		{filePath: 'src/com/radaee/util/CommonUtil.java', importStatement: 'com.radaee.viewlib.R'},
-		{filePath: 'src/com/radaee/util/RadaeePDFManager.java', importStatement: 'com.radaee.viewlib.R'},
-		{filePath: 'src/com/radaee/util/BookmarkHandler.java', importStatement: 'com.radaee.viewlib.R'}
+	{filePath: `${appSrcJavaDir}/com/radaee/cordova/RadaeePDFPlugin.java`, importStatement: 'com.radaee.viewlib.R'},
+    	{filePath: `${appSrcJavaDir}/com/radaee/pdf/Global.java`, importStatement: 'com.radaee.viewlib.R'},
+    	{filePath: `${appSrcJavaDir}/com/radaee/reader/PDFLayoutView.java`, importStatement: 'com.radaee.viewlib.R'},
+    	{filePath: `${appSrcJavaDir}/com/radaee/reader/PDFNavAct.java`, importStatement: 'com.radaee.viewlib.R'},
+    	{filePath: `${appSrcJavaDir}/com/radaee/reader/PDFViewAct.java`, importStatement: 'com.radaee.viewlib.R'},
+    	{filePath: `${appSrcJavaDir}/com/radaee/reader/PDFViewController.java`, importStatement: 'com.radaee.viewlib.R'},
+	{filePath: `${appSrcJavaDir}/com/radaee/reader/PDFPagerAct.java`, importStatement: 'com.radaee.reader.R'},
```

Comment:
Inconsistant indents.

JS files that use `var` are usually trying to be ES5 compatible, so I wouldn't start using template strings here.